src/components/Payment.js

```diff
@@ -21,17 +40,26 @@ const CashPayment = ({ onPaymentSelect, selectedValue }) => (
           inputProps={{ 'aria-label': 'Cash' }}
         />
 )}
-      label="Cash"
+      label={(
+        <div className="payment-label">
+          <div className="img-style">
```

Comment:
Rename this with payment-img-style ; Something like what less css would do

Reply:
Updated